.travis.yml

```diff
@@ -144,6 +145,10 @@ install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
 
+    # Use this until https://github.com/sphinx-gallery/sphinx-gallery/pull/255
+    # is merged.
+    - if [[ $SETUP_CMD == *build_docs* ]]; then pip -q install git+https://github.com/pllim/sphinx-gallery.git@stat-iter-deprec#egg=sphinx-gallery; fi
```

Comment:
does this warning cause a sphinx failure? if it does, could we maybe use the functionality of https://github.com/astropy/astropy/pull/6223 to turn it off so not just travis, but local builds would pass without issues?